src/Application/Redux/actions.js

```diff
@@ -0,0 +1,34 @@
+export const CLICK_MONEY = 'CLICK_MONEY';
+export const COLLECT_INCOME = 'COLLECT_INCOME';
+export const BUY_ASSET = 'BUY_ASSET';
+export const UNLOCK_ASSET = 'UNLOCK_ASSET';
+export const UPGRADE_CURRENCY = 'UPGRADE_CURRENCY';
+export const BROADCAST_NEWS = 'BROADCAST_NEWS';
```

Comment:
Seems reasonable, I think at some point I will rework the actions file and I will try to keep this in mind